.travis.yml

```diff
@@ -54,7 +54,7 @@ install:
   - luarocks make
   - cd kong-ce
   # the following line is a hack to get around failing LuaSec upgrades
-  - luarocks remove luasec --force
+  #- luarocks remove luasec --force
```

Comment:
Can we remove these lines if they are no longer required?

Reply:
This can be done in a separate PR. Going to create an issue to follow up on this, and we can discuss there. https://github.com/Kong/kong-plugin-serverless-functions/issues/18